index.ts

```diff
@@ -26,14 +26,21 @@ export const background = rootContext();
 export const TODO = rootContext();
 
 export const requestIdKey = Symbol("requestId");
-
 export function getRequestId(ctx: Context): string | undefined {
   if (requestIdKey in ctx) {
     // typescript doesn't like symbols, but it's safe
     return String((ctx as Record<typeof requestIdKey, unknown>)[requestIdKey]);
   }
 }
 
+export const actorKey = Symbol("actor");
+export function getActor(ctx: Context): string | undefined {
+  if (actorKey in ctx) {
+    // typescript doesn't like symbols, but it's safe
+    return String((ctx as Record<typeof actorKey, unknown>)[actorKey]);
```

Comment:
Can see the other getter does this anyway so wont request anything.

But feel like this could do a typeof string check instead of forcing whatever is set here into a string which could result in `Object object`